test/linear_combination.lean

```diff
@@ -1,6 +1,10 @@
-import Mathlib.Data.Real.Basic
 import Mathlib.Tactic.LinearCombination
 
+-- We deliberately mock R here so that we don't have to import the deps
+axiom Real : Type
+notation "ℝ" => Real
+@[instance] axiom Real.linearOrderedRing : LinearOrderedField ℝ
```

Comment:
could the name be coherent with the content? (I don't know if you want a field or a ring)